python/py-tensorflow_macos/Portfile

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        apple tensorflow_macos 0.1 v alpha3
+name                py-${github.project}
+version             ${github.version}.${github.tag_suffix}
+revision            0
+
+platforms           darwin
+categories-append   lang
+license             restrictive/distributable
+maintainers         nomaintainer
+
+description         Mac-optimized TensorFlow and TensorFlow Addons
+
+long_description    Hardware-accelerated TensorFlow and TensorFlow \
+                    Addons for macOS 11.0+. Native hardware \
+                    acceleration is supported on M1 Macs and \
+                    Intel-based Macs through Apple’s ML Compute \
+                    framework.
+
+# https://github.com/apple/tensorflow_macos/blob/master/scripts/download_and_install.sh
+distfiles           ${github.project}-${github.version}${extract.suffix}:tarball \
+                    ${github.project}-${github.version}${github.tag_suffix}${extract.suffix}:releases
+
+master_sites        ${github.homepage}/tarball/${git.branch}:tarball \
+                    github.master_sites ${github.homepage}/releases/download/${git.branch}:releases
+
+checksums           [lindex [split [lindex ${distfiles} 0] :] 0] \
+                    rmd160  975a2ee2508df2fe801b841d3f6f8a0172854292 \
+                    sha256  623237c06941d26f52280535c0139c9d58fc214e0378b4513803a65bfda48ff6 \
+                    size    8095 \
+                    [lindex [split [lindex ${distfiles} 1] :] 0] \
+                    rmd160  812afc151dd87abb657df6d8e66ad8681c91a3bd \
+                    sha256  80807f8b0fc4a98ffa0b7395304fa61239f246a84f1eedbca70a5a1847d4dead \
+                    size    376699291
+
+extract.cmd         ${portutil::autoconf::tar_command}
+extract.pre_args    -xzf
+extract.post_args
+
+if {${os.platform} eq "darwin" && ${os.major} < 20} {
+    known_fail      yes
+    pre-fetch {
+        ui_error "TensorFlow with ML Compute acceleration is only available \
+            on macOS 11.0 and later."
+        error {unsupported platform}
+    }
+}
+
+python.versions 38
```

Comment:
Does this new port not support python 3.9 as well at least ?

Reply:
Nope. No source code,.No Python 39. Yet. Just pre-compiled binaries for CPython 38.